hudi-client/src/test/java/org/apache/hudi/index/bloom/TestHoodieBloomIndex.java

```diff
@@ -353,8 +352,8 @@ public void testTagLocation() throws Exception {
         HoodieClientTestUtils.writeParquetFile(basePath, "2015/01/31", Arrays.asList(record4), schema, null, true);
 
     // We do the tag again
-    metadata = new HoodieTableMetaClient(jsc.hadoopConfiguration(), basePath);
```

Comment:
I kind of prefer to have it be a new instance? instead of introducing a reload method.. Those things tend to get tricky over time (someone forget to reset one variable etc).. wdyt?